spec/src/main/asciidoc/core/events.asciidoc

```diff
@@ -317,6 +320,24 @@ public void afterDocumentUpdated(@Observes @Updated Document doc) { ... }
 public void afterDocumentEvent(@Observes Document doc) { ... }
 ----
 
+On the other hand, following observer will not be notified as slightly different behaviour applies to observers with `@Default` qualifier:
+
+[source, java]
+----
+public void afterDocumentDefaultEvent(@Observes @Default Document doc) { ... }
+----
+
+Such observer will only be notified for events having either no qualifiers or only `@Default` qualifier:
```

Comment:
This is nothing new. It is how it works ATM.
From specification, [section 2.3.1](http://docs.jboss.org/cdi/spec/2.0.EDR1/cdi-spec.html#builtin_qualifiers): _The default qualifier is also assumed for any injection point that does not explicitly declare a qualifier, as defined in The default qualifier at injection points._

This then points to [section 3.8](http://docs.jboss.org/cdi/spec/2.0.EDR1/cdi-spec.html#injection_point_default_qualifier) where there is an example:

> Finally, the following are equivalent:
> `@Inject Order order;`
> `@Inject @Default Order order;`